apps/meteor/app/lib/server/lib/notifyListener.ts

```diff
@@ -467,3 +470,119 @@ export const notifyOnMessageChange = withDbWatcherCheck(async ({ id, data }: { i
 	}
 	void api.broadcast('watch.messages', { message });
 });
+
+export const notifyOnSubscriptionChanged = withDbWatcherCheck(
+	async (subscription: ISubscription, clientAction: ClientAction = 'updated'): Promise<void> => {
+		void api.broadcast('watch.subscriptions', { clientAction, subscription });
+	},
+);
+
+export const notifyOnSubscriptionChangedByRoomIdAndUserId = withDbWatcherCheck(
+	async (
+		rid: ISubscription['rid'],
+		uid: ISubscription['u']['_id'],
+		clientAction: Exclude<ClientAction, 'removed'> = 'updated',
+	): Promise<void> => {
+		const subscriptions = Subscriptions.findByUserIdAndRoomIds(uid, [rid], { projection: subscriptionFields });
+
+		for await (const subscription of subscriptions) {
```

Comment:
```suggestion
		for (const subscription of subscriptions) {
```

Reply:
this is because of the AsyncIterator.. but in fact, we don't need to use it